tests/ondemand/CMakeLists.txt

```diff
@@ -1,15 +1,13 @@
-# All remaining tests link with simdjson proper
 link_libraries(simdjson)
 include_directories(..)
-add_subdirectory(compilation_failure_tests)
 
 add_cpp_test(ondemand_active_tests           LABELS ondemand acceptance per_implementation)
 add_cpp_test(ondemand_array_tests            LABELS ondemand acceptance per_implementation)
 add_cpp_test(ondemand_array_error_tests      LABELS ondemand acceptance per_implementation)
 add_cpp_test(ondemand_compilation_tests      LABELS ondemand acceptance per_implementation)
 add_cpp_test(ondemand_error_tests            LABELS ondemand acceptance per_implementation)
 add_cpp_test(ondemand_key_string_tests       LABELS ondemand acceptance per_implementation)
-add_cpp_test(ondemand_misc_tests           LABELS ondemand acceptance per_implementation)
+add_cpp_test(ondemand_misc_tests            LABELS ondemand acceptance per_implementation)
```

Comment:
You should consider adding two spaces instead of one. This is literally only half a fix!